docs/blog/posts/on-dog-food-the-original-metaverse-and-not-being-bored.md

```diff
@@ -288,7 +288,7 @@ So, thanks to this bit of code in my `Activity` widget...
             parent.move_child(
                 self, before=parent.children.index( self ) - 1
             )
-            self.emit_no_wait( self.Moved( self ) )
+            self.post_messa_no_wait( self.Moved( self ) )
```

Comment:
Dear diary, today I found a typo in Rodrigo's code. Today was a good day.

Reply:
Fixed in c39b23f.